app/networkapi/static/highlights/js/admin.js

```diff
@@ -0,0 +1,13 @@
+function run($) {
+  var fieldsTiedToFeaturedCards = ".quote, .bio, .interview_url";
+
+  $(".featured").on('click', function() {
+    if(!$("#id_featured").is(':checked')) {
```

Comment:
can `$("#id_featured")` be cached so it's not constantly re-retrieved during click handling?